var/spack/repos/builtin/packages/pythia8/package.py

```diff
@@ -31,50 +31,95 @@ class Pythia8(AutotoolsPackage):
     version('8.212', sha256='f8fb4341c7e8a8be3347eb26b00329a388ccf925313cfbdba655a08d7fd5a70e', deprecated=True)
 
     variant('shared', default=True, description='Build shared library')
-    variant('hepmc', default=True, description='Build HepMC2 extensions')
-    variant('evtgen', default=False, description='Build EvtGen extensions')
-    variant('root', default=False, description='Build ROOT extensions')
-    variant('fastjet', default=False, description='Build fastjet extensions')
+    variant('hepmc', default=True, description='Export PYTHIA events to the HEPMC format, version 2')
+    variant('hepmc3', default=True, description='Export PYTHIA events to the HEPMC format, version 3')
+    variant('evtgen', default=False, description='Particle decays with the EvtGen decay package')
+    variant('root', default=False, description='Use ROOT trees and histograms with PYTHIA')
+    variant('fastjet', default=False, description='Building of jets using the FastJet package, version 3')
+    variant('lhapdf', default=False, description='Support the use of external PDF sets via LHAPDF')
+    variant('rivet', default=False, description='Support use of RIVET through direct interface')
+    variant('python', default=False, description='Interface to use PYTHIA in Python')
+    variant('madgraph5amc', default=False, description='MadGraph matrix element plugins for parton showers')
+    variant('openmpi', default=False, description='Multi-threading support via OpenMP')
+    variant('mpich', default=False, description='Multi-threading support via MPICH')
+    variant('hdf5', default=False, description='Support the use of HDF5 format')
 
     depends_on('rsync', type='build')
-    depends_on('hepmc@:2.99.99', when="+hepmc")
-    depends_on('root', when="+root")
-    depends_on('evtgen', when="+evtgen")
-    depends_on("fastjet@3.0.0:", when="+fastjet")
-
-    conflicts("^evtgen+pythia8", when="+evtgen",
-              msg="Building pythia with evtgen bindings and "
-              "evtgen with pythia bindings results in a circular dependency "
-              "that cannot be resolved at the moment! "
-              "Use pythia8+evtgen^evtgen~pythia8")
+    depends_on('hepmc@:2.99.99', when='+hepmc')
+    depends_on('hepmc@3:', when='+hepmc3')
```

Comment:
Does this actually build in the default configuration?

```python
variant('hepmc', default=True, …)
variant('hepmc3', default=True, …)
```

I wonder how it can satisfy `hepmc@:2.99.99` and `hepmc@3:`?

Reply:
Thanks, that's a typo